packages/react-ui-core/src/Modal/__tests__/Modal-test.js

```diff
@@ -20,25 +20,25 @@ describe('Modal/Modal', () => {
       wrapper = shallow(<Modal theme={theme} onClose={onClose} />)
     })
 
-    test('adds `className` prop to the root node', () => {
```

Comment:
Not really needed to change this here as they're both synonymous.  Either way it's fine for me though.

Reply:
More a matter of standardizing as everywhere else uses `it`.